bluemira/equilibria/flux_surfaces.py

```diff
@@ -850,12 +856,18 @@ def calculate_connection_length_fs(
     passing through Coils, but really they should be intercepted beforehand!
     """
     if first_wall is None:
-        x1, x2 = eq.grid.x_min, eq.grid.x_max
-        z1, z2 = eq.grid.z_min, eq.grid.z_max
+        x1, x2 = eq.grid.x_min + 0.01, eq.grid.x_max - 0.01
+        z1, z2 = eq.grid.z_min + 0.01, eq.grid.z_max - 0.01
         first_wall = Coordinates({"x": [x1, x2, x2, x1, x1], "z": [z1, z1, z2, z2, z1]})
 
-    xfs, zfs = find_flux_surface_through_point(eq.x, eq.z, eq.psi(), x, z, eq.psi(x, z))
-    f_s = OpenFluxSurface(Coordinates({"x": xfs, "z": zfs}))
+    if f_s is None:
```

Comment:
Could we test for fs == None, fs.closed would be nice too but thats much simpler

Reply:
Have added tests for both.